opentelemetry-api/src/opentelemetry/configuration/__init__.py

```diff
@@ -101,6 +104,11 @@ def __new__(cls) -> "Configuration":
 
                     key = match.group(1)
 
+                    if value == "True":
+                        value = True
+                    if value == "False":
+                        value = False
+
```

Comment:
What about integer / floats types? should we perform a similar casting?

Reply:
Fixed